kview/src/main/java/dev/ikm/komet/kview/mvvm/view/genediting/GenEditingDetailsController.java

```diff
@@ -460,22 +463,16 @@ private void setupSemanticDetails() {
                         }
                     }
                 }
-
             }
 
             semanticEntityVersionLatest = retrieveCommittedLatestVersion(observableSemanticSnapshot);
             //Set and Update STAMP values
             semanticEntityVersionLatest.ifPresent(semanticEntityVersion -> {
-//                StampEntity stampEntity = semanticEntityVersion.stamp();
-//                stampViewModel.setPropertyValue(STATUS, stampEntity.state())
-//                        .setPropertyValue(TIME, stampEntity.time())
-//                        .setPropertyValue(AUTHOR, stampEntity.author())
-//                        .setPropertyValue(MODULE, stampEntity.module())
-//                        .setPropertyValue(PATH, stampEntity.path())
-//                ;
-//                stampViewModel.save(true);
+                propertiesController.getStampFormViewModel().setPropertyValue(CURRENT_STAMP, semanticEntityVersion.stamp());
```

Comment:
It's not clear in the view model classes but we shouldn't be updating the CURRENT_STAMP property outside of the view model. 
CURRENT_STAMP should be a read-only property that only gets updated by the view model itself (encapsulation).
If you call propertiesController.getStampFormViewModel().update(...) it should update all the view model properties appropriately.